BackEnd/DockerREADME.md

```diff
@@ -2,33 +2,28 @@
 **With Docker**
 ```
 # to build the admin core
-docker build -f docker/admin.core.Dockerfile -t unosquare/admincore:0.0.1 .
-docker tag docker/admincore:0.0.1 docker/admincore:latest
+docker build -f docker/admin.core.Dockerfile -t unosquare/admincore:<version> .
+docker tag docker/admincore:<version> docker/admincore:latest
```

Comment:
I think this should be unosquare/admincore here.

Reply:
yep, you are right